tests/phpunit/CRM/Core/TransactionTest.php

```diff
@@ -364,7 +362,7 @@ public function createContactWithTransaction($nesting, $insert, $outcome) {
 
     if ($insert == 'sql-insert') {
       $r = CRM_Core_DAO::executeQuery("INSERT INTO civicrm_contact(first_name,last_name) VALUES ('ff', 'll')");
-      $cid = mysql_insert_id();
+      $cid = CRM_Core_DAO::singleValueQuery("SELECT max(id) FROM civicrm_contact");
```

Comment:
@totten can you review this to see if you think this is ok replacement?

Reply:
Good question, @seamuslee001. If there were more concurrency, then it could be suspicious, but there isn't. IMHO, if the tests pass, then it's probably OK.